src/main/kotlin/com/exactpro/th2/read/file/common/state/ReaderState.kt

```diff
@@ -49,5 +48,5 @@ data class StreamData(
     /**
      * The content of the last message in [StreamId]
      */
-    val lastContent: ByteString,
+    val lastContent: Any,
```

Comment:
I don't think that we should use that... Can we use ByteArray instead? Or can we provide an interface that will provide the required API?

Reply:
If using ByteArray it will cause unnecessary convertion ByteString to ByteArray. We only using .equals() method on this 'lastContent' object, and in this case 'Any' can be treated as such interface with required API.

Reply:
Using `Any` does not help to understand what the content is... I insist that we should create a sealed interface or class that will encapsulate the content type and provide you with required method

Reply:
@lumber1000 Do you have any updates on this?